### DISCUSSION TITLE
buddhism/2024/heart_sutra_memorization/

### DISCUSSION THREAD

**Discussion by giscus** [shane0/buddhism#112] [General]
# buddhism/2024/heart_sutra_memorization/

https://shanenull.com/buddhism/2024/heart_sutra_memorization/

**Comment by shane0** [comment]
https://youtu.be/3BPLhSWSYIE?si=RfLms38hU2NeVP0T

**Reply by shane0** [reply]
redo audio is low and add logger and read a file instead of a python list
also mention further study
https://shanenull.com/buddhism/2024/prajna_paramita/

**Comment by shane0** [comment]
Commentary book
https://github.com/shane0/buddhism/issues/113

**Comment by shane0** [comment]
Show all button
Touch anywhere
Button on bottom
